lib/liveValidation/operationValidator.ts

```diff
@@ -358,21 +357,11 @@ const validateLroHeader = (
   operation: Operation,
   statusCode: string,
   headers: StringMap<string>,
-  body: any,
   result: LiveValidationIssue[]
 ) => {
-  if (statusCode === "201" && body?.properties !== undefined) {
-    const properties = body.properties;
-    if (
-      properties &&
-      (properties.provisioningState === undefined ||
-        properties.provisioningState === "Succeeded" ||
-        properties.provisioningState === "Failed" ||
-        properties.provisioningState === "Canceled")
-    ) {
-      // Ignore LRO header check when it's sync call
-      return;
-    }
+  if (statusCode === "201") {
+    // Ignore LRO header check cause RPC says azure-AsyncOperation is optional if using 201/200+ provisioningState
```

Comment:
What if statusCode is  202?

Reply:
Yes, 202 case has to provide LRO headers.